### PR TITLE
Toolbar

### PR DESCRIPTION
**Pull request by vxsx**

### CODE REVIEW

**Comment by mkoistinen** [comment]
lgtm

File: cms/static/cms/sass/components/_toolbar.scss [hunk 1]
```diff
@@ -307,94 +292,98 @@
...
         .cms-error input {
-            border: 1px solid $color-red;
+            border: 1px solid $color-danger;
         }
     }
 
     // #TOOLBAR/elements/buttongroups#
     .cms-toolbar-item-buttons {
-        margin: 4px 0 4px;
+        $margin: ($toolbar-height - $toolbar-button-height) / 2;
```
Comment (FinalAngel) [inline cms/static/cms/sass/components/_toolbar.scss]: will that only be a local variable?

File: cms/templates/cms/toolbar/toolbar.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% load i18n l10n %}
 
 {% language request.toolbar.toolbar_language %}
-<div id="cms-toolbar" class="cms-reset{% if not user.is_authenticated %} cms-toolbar-auth{% endif %}{% if debug %} cms-toolbar-debug{% endif %}" dir="ltr">
+<div id="cms-toolbar" class="cms-reset{% if not user.is_authenticated %} cms-toolbar-auth{% endif %}{% if True or debug %} cms-toolbar-debug{% endif %}" dir="ltr">
```
Comment (FinalAngel) [inline cms/templates/cms/toolbar/toolbar.html]: why the true here?

**Comment by FinalAngel** [comment]
LGTM except for the comments

File: cms/templates/cms/toolbar/toolbar.html (hunk 2 above)
Comment (vxsx) [inline cms/templates/cms/toolbar/toolbar.html]: oops, that was not supposed to be here

File: cms/static/cms/sass/components/_toolbar.scss (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/sass/components/_toolbar.scss]: yes
```
Variables are only available within the level of nested selectors where they’re defined. 
```
from http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variables_

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**